pkg/webhook/utils/in_capsule_group.go

```diff
@@ -26,15 +26,15 @@ import (
 	"github.com/clastix/capsule/pkg/webhook"
 )
 
-func InCapsuleGroup(capsuleGroup string, webhookHandler webhook.Handler) webhook.Handler {
+func InCapsuleGroup(capsuleGroup []string, webhookHandler webhook.Handler) webhook.Handler {
 	return &handler{
 		handler:      webhookHandler,
 		capsuleGroup: capsuleGroup,
 	}
 }
 
 type handler struct {
-	capsuleGroup string
+	capsuleGroup []string
```

Comment:
Just a naming issue: maybe better `capsuleGroups`?

Reply:
Yep, thank you. Renamed